api/types/github.go

```diff
@@ -279,12 +279,12 @@ func (c *GithubConnectorV3) SetDisplay(display string) {
 
 // GetEndpointURL returns the endpoint URL
 func (c *GithubConnectorV3) GetEndpointURL() string {
-	return githubURL
+	return GithubURL
```

Comment:
Is anyone else curious why this always returns a constant? Is this because SSO for self-hosted github is an enterprise-only feature?